File: Java/Lista_04(Vetores)/Alturas.java
import java.util.Scanner;
import java.util.Locale;

public class Alturas {

    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner s = new Scanner(System.in);

        System.out.print("Quantas pessoas serao digitadas? ");
        int n = s.nextInt();

        String nome[] = new String[n];
        int idade[] = new int[n];
        double altura[] = new double[n];

        for (int i = 0; i < nome.length; i++) {
            System.out.println("Dados da " + (i + 1) + "a pessoa:");
            System.out.print("Nome: ");
            s.nextLine();
            nome[i] = s.nextLine();
            System.out.print("Idade: ");
            idade[i] = s.nextInt();
            System.out.print("Altura: ");
            altura[i] = s.nextDouble();
        }

        double soma = 0;

        for (int i = 0; i < altura.length; i++) {
            soma += altura[i];
        }

        double media = soma / altura.length;

        System.out.printf("Altura media: %.2f\n", media);

        int cont = 0;

        for (int i = 0; i < idade.length; i++) {
            if (idade[i] < 16) {
                cont++;
            }
        }

        double por = cont * 100 / idade.length;

        System.out.printf("Pessoas com menos de 16 anos: %.1f %%\n", por);

        for (int i = 0; i < idade.length; i++) {
            if (idade[i] < 16) {
                System.out.println(nome[i]);
            }
        }

        s.close();

    }
}

File: Python/Lista_04(Vetores)/Alturas.py
n = int(input("Quantas pessoas serao digitadas? "))

nome = [0 for x in range(n)]
idade = [0 for x in range(n)]
altura = [0 for x in range(n)]

for i in range(0,n):
    print(f"Dados da {i+1}a pessoa:")
    nome[i] = input("Nome: ")
    idade[i] = int(input("Idade: "))
    altura[i] = float(input("Altura: "))

soma = 0.0

for i in range(0,n):
    soma += altura[i]

media = soma/n

print(f"\nAltura media: {media:.2f}")

cont = 0

for i in range(0,n):
    if idade[i]<16:
        cont += 1

per = cont/n*100

print(f"Pessoas com menos de 16 anos: {per:.1f}%")

for i in range(0,n):
    if idade[i]<16:
        print(nome[i])